backend/app/models/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .registration import Registration
 from .role import Role
 from .staff import Staff
+from .skill import Skill
```

Comment:
Need EOF blank line (i.e. one empty line after) -- install the pre-commit hooks, it should automatically check for you

Reply:
Why do we need this ah? Is this for just style guide or there's semantic value

Reply:
I searched up why too and just did it for all files

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Reply:
^ pretty much summarizes why, but its an in-built styling pattern applied in Python Black